src/configuration/configuration.ts

```diff
@@ -203,6 +203,7 @@ class Configuration implements IConfiguration {
   easymotionMarkerFontWeight = 'normal';
   easymotionMarkerYOffset = 0;
   easymotionKeys = 'hklyuiopnm,qwertzxcvbasdgjf;';
+  easymotionJumpToAnywhereRegex = '';
```

Comment:
Let's set the default here: `\\b[A-Za-z0-9]|[A-Za-z0-9]\\b|_.|#.|[a-z][A-Z]`.

Reply:
You'll also need to update the `package.json` to add this value. Check out the comment at the top of this class.

Reply:
@jpoon Great idea. Much cleaner logic in `easymotion.cmd.ts` now :)